frontend/src/domains/maps/hooks/useKakaoMapSDK.ts

```diff
@@ -1,24 +1,43 @@
-import { useEffect, useState } from 'react';
-
+import { useEffect, useRef, useState } from 'react';
 import type { UseKakaoMapSDKReturn } from '../types/KaKaoMap.types';
 
 export const useKakaoMapSDK = (): UseKakaoMapSDKReturn => {
   const [sdkReady, setSdkReady] = useState(false);
   const [sdkError, setSdkError] = useState<string | null>(null);
 
+  const timersRef = useRef<Map<string, number>>(new Map());
```

Comment:
map으로 여러개의 타이머를 관리할 필요가 없을 것 같은데 어떻게 생각하시나요??

```suggestion
  const timersRef = useRef: number | null(null);
```

토스트처럼 여러 타이머가 동시에 개별적으로 돌아가야 하는 상황은 아니라고 생각됩니다!